frontend/src/app/main/data/team.cljs

```diff
@@ -255,12 +255,12 @@
   (dm/assert! (string? name))
   (ptk/reify ::create-team
     ptk/WatchEvent
-    (watch [it state _]
+    (watch [it _ _]
       (let [{:keys [on-success on-error]
              :or {on-success identity
                   on-error rx/throw}} (meta params)
-            features (features/get-enabled-features state)
```

Comment:
I can't see how `features/global-enabled-features` is returning the same values as `features/get-enabled-features`

Reply:
They don't return the exact the same, but for team creation that difference has no real difference.

Reply:
Ok, I'll take it with faith :D